app/main.py

```diff
@@ -1,6 +1,34 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    params = {
+        "key": api_key,
+        "q": os.getenv("CITY_NAME")
```

Comment:
As mentioned in the previous comment, consider defining a constant for the city name. You can fetch the city name from the environment variables once and assign it to a constant. Then, use this constant in your code.